src/app/work-item/work-item-quick-add/work-item-quick-add.component.html

```diff
@@ -10,14 +10,14 @@
     </div>
     <div class="list-group-item-text margin10">
       <div class="form-group">
-        <input [(ngModel)]="workItem.fields['system.description']" class="form-control" placeholder="Description" />
+        <input [(ngModel)]="workItem.fields['system.description']" (keypress)="eventHandler($event)" class="form-control" placeholder="Description" />
       </div>
     </div>
     <div class="list-group-item-text margin10">
       <div class="form-group">
-        <a class='pficon-add-circle-o' [ngClass]="{'icon-btn': validName,'icon-btn-disabled':!validName}" (click)="save()"></a>&nbsp;&nbsp;
+        <a class='pficon-add-circle-o' [ngClass]="validName ? 'icon-btn' : 'icon-btn-disabled'" (click)="save()"></a>&nbsp;&nbsp;
```

Comment:
Actually in later commit use of validName is removed. That action is regarding setting up test. In order to setup test I had to refactor the code with Angular2's specification.